bitnami/solr/templates/statefulset.yaml

```diff
@@ -186,8 +186,8 @@ spec:
             - name: SOLR_COLLECTION
               value: {{ .Values.collection | quote }}
             {{- else }}
-            - name: SOLR_CORE
-              value: {{ .Values.coreName | quote }}
+            - name: SOLR_CORES
+              value: {{ .Values.coreNames | join "," | quote }}
```

Comment:
Could we add a backwards with `coreName` (we don't have to add it in values.yaml) so we don't have to force a major bump?

Reply:
@javsalgar I don't mind implementing a solution as you suggested (actually this was my initial implementation).
But I'm wondering if that's a good idea. I mean, the chart is currently broken concerning this very particular feature.
So users will have to upgrade their chart to a new version: why couldn't they take this occasion to rename `coreName` to `coreNames` in their system (and pass it as an array)?
And, again, this change is introduced in version `2.2.0`, i.e. a new minor version.

Back to your proposal: should `coreName` coexist next to `coreNames`, when should the first disappear? In other words, shouldn't `coreName` be documented as *deprecated*, stating when it will be removed altogether?

Reply:
Hi,

I agree that it's a good idea to show the value as deprecated, and maybe show a warning in the NOTES.txt file so the user gets a warning saying that the value is to be deprecated.

I would remove the value in the next major version.

Reply:
I've made some changes as requested in order to be backward compatible.